src/main/kotlin/com/mparticle/kits/AppboyKit.kt

```diff
@@ -97,6 +98,12 @@ open class AppboyKit : KitIntegration(), AttributeListener, CommerceListener,
             )
         }
         setIdentityType(settings)
+
+        val user = MParticle.getInstance()!!.Identity().currentUser
```

Comment:
Though unlikely, if getInstance is ever null, this will crash...

Since you're checking if user is null anyway, it would be better to use safe unwrapping like we would in Swift:

```suggestion
        val user = MParticle.getInstance()?.Identity()?.currentUser
```

Double check that the ? is needed after Identity(), if not, remove it and just use the ? after getInstance()